nodes/src/nodes/llm_base/streaming.py

```
@@ -0,0 +1,377 @@
+    def _send_sse(self, event_type: str, **data: Any) -> None:
+        """Dispatch an SSE event via the engine's ``sendSSE`` interface.
+
+        Silently swallows errors so that SSE failures never break the
+        main response flow.
+        """
+        try:
+            self._instance.instance.sendSSE(event_type, **data)
+        except Exception:
+            # SSE delivery is best-effort; never let it break the
+            # chat response path.
+            pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)